object-store-client-play-26/src/test/scala/uk/gov/hmrc/objectstore/client/play/PlayObjectStoreClientEitherSpec.scala

```diff
@@ -337,64 +355,4 @@ class PlayObjectStoreClientEitherSpec
     """{
       |    "objects": []
       |}""".stripMargin
-
-  private def initPutObjectStub(path: Path.File, statusCode: Int, reqBody: Array[Byte], md5Base64: String): Unit = {
```

Comment:
I had actually envisioned refactoring the duplication between PlayObjectStoreClientEitherSpec and PlayObjectStoreClientSpec in a different way - given that the tests are all effectively the same (and the stubs just support those tests, so can stay where they are).
e.g. make extension points for the differences - e.g. client construction, and result assertion